Lmod/branches/7.0/z01-default_modules.sh

```diff
@@ -0,0 +1,11 @@
+if [ -z "$__Init_Default_Modules" -o -z "$LD_LIBRARY_PATH" ]; then
+   export __Init_Default_Modules=1;
+   if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
+     if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
+       export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
```

Comment:
Hi, 1) `/usr/share/modulefiles/DefaultModules.lua` does not necessarily exist across all systems,
`LMOD_SYSTEM_DEFAULT_MODULES` should be tunable regardless of that.
2) there is missing configuration for .csh users; we don't have many of them but we need have them complain either